src/assets/js/protocol/protocol-newsletter.js

```diff
@@ -0,0 +1,41 @@
+(function() {
+    'use strict';
+
+    // !! This file assumes only one signup form per page !!
+
+    // Expand email form on input focus or submit if details aren't visible
+    function initEmailForm() {
+        var newsletterForm = document.getElementById('newsletter-form');
+        var submitButton = document.getElementById('newsletter-submit');
+        var formDetails = document.getElementById('newsletter-details');
+        var formControls = document.querySelectorAll('.mzp-c-newsletter-form input, .mzp-c-newsletter-form select');
+
+        function emailFormShowDetails() {
+            if (window.getComputedStyle(formDetails).display === 'none') {
```

Comment:
Rather than use these`getComputedStyle` calls, it may be slightly more performant to use a boolean variable to determine if the form has expanded? e.g. 

```javascript
if (!formExpanded) {
    formDetails.style.display = 'block';
    formExpanded = true;
}
```